TS/oshinoko/app/ui/VideoUpload.tsx

```diff
@@ -2,13 +2,17 @@
 import { useState } from 'react';
 import axios from 'axios';
 import { backendUrl } from '@/app/lib/config';
+import EnhancedVideoPlayer from '@/app/ui/EnhancedVideoPlayer';
 
 const VideoUpload = () => {
   const [message, setMessage] = useState<string>('');
   const [videoData, setVideoData] = useState<{
     title: string;
     group_name: string;
     video_url: string;
+    overlay_url: string;
+    original_video_width: number;
+    original_video_height: number;
   } | null>(null);
```

Comment:
```suggestion
  const [videoData, setVideoData] = useState<Video | null>(null);
```
上にimport をついか
`import { Video } from '@/app/lib/types';`
あとはフォーマットをお願いします。

Reply:
id はnullableにしてないからそこの処理も必要